spark/src/main/scala/org/apache/spark/sql/delta/managedcommit/InMemoryCommitStore.scala

```diff
@@ -126,6 +126,16 @@ class InMemoryCommitStore(val batchSize: Long) extends AbstractBatchBackfillingC
       versionsToRemove.foreach(tableData.commitsMap.remove)
     }
   }
+
+  def registerTable(
+      logPath: Path,
+      maxCommitVersion: Long): Unit = {
+    val newPerTableData = new PerTableData()
+    newPerTableData.maxCommitVersion = maxCommitVersion
```

Comment:
nit: any reason this isn't a constructor arg?

Reply:
This is a class and not a case class. So constructor is not defined.

Reply:
```scala
class Foo(var x: Int = -1)
val f = new Foo(10)
f.x += 10
println(f.x)
```
Works just fine and prints `20`?

(case classes just define `apply` and `unapply` helpers automatically)